diff --git a/node_modules/@yao-pkg/pkg/prelude/bootstrap.js b/node_modules/@yao-pkg/pkg/prelude/bootstrap.js
index 5fc3ee6..e3f822f 100644
--- a/node_modules/@yao-pkg/pkg/prelude/bootstrap.js
+++ b/node_modules/@yao-pkg/pkg/prelude/bootstrap.js
@@ -1046,8 +1046,9 @@ function payloadFileSync(pointer) {
 
     const entityLinks = entity[STORE_LINKS];
     if (entityLinks) return cb2(error_EISDIR(path_));
-
+    console.log(entity);
     const entityContent = entity[STORE_CONTENT];
+    console.log(entityContent);
     if (entityContent) return readFileFromSnapshotSub(entityContent, cb);
 
     const entityBlob = entity[STORE_BLOB];
